.github/workflows/build.yaml

```diff
@@ -472,3 +511,23 @@ jobs:
             tag=${{ env.BRANCH }}
             dockerRepository=${{ env.REGISTRY }}
             platform=x86_64
+
+  trigger-workflow:
+    name: Trigger workflow depending upon the branch
+    needs: [build-obr, build-obr-generic, build-obr-javadocs]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Triggering helm build (github.ref is main)
+        if: env.BRANCH == 'main'
+        env:
+          GH_TOKEN: ${{ secrets.GALASA_TEAM_GITHUB_TOKEN }}
+        run: |
+         gh workflow run build-helm.yaml --repo https://github.com/galasa-dev/automation --ref ${{ env.BRANCH }}

```

Comment:
This is great thanks - however for now can we comment out this step please? Until we fully switch over to GitHub actions, we don't want this to get triggered, as the Tekton equivalent will be triggered at a similar time, and both Helm workflows running in parallel will cause issues due to the fact it uninstalls then installs our 'Galasa ecosystem'.

Reply:
done